### PR TITLE
Fix/extract testing environment setup functions to package called internal-test-env

### PR DESCRIPTION
**Pull request by chelseapinka**
# New module "internal-test-env" for repeatable test setup

# Checklist

- [x] All acceptance criteria are met.
- [x] Relevant documentation, if any, has been written/updated.
- [x] The changelog has been updated, if applicable.
- [x] New functions/types have been exported in `index.ts`, if applicable.
- [x] New modules (i.e. new `.ts` files) are listed in the `exports` field in `package.json`, if applicable.
- [x] New modules (i.e. new `.ts` files) are listed in the `typedocOptions.entryPoints` field in `tsconfig.json`, if applicable.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: e2e/browser/test/e2e.playwright.ts [hunk 1]
```diff
@@ -20,11 +20,14 @@
...
+import {
+  setupEnv,
+  getBrowserTestingEnvironment,
+} from "@inrupt/test-env-helpers";
 import { essUserLogin } from "./roles";
 
-import { getTestingEnvironmentBrowser } from "../../e2e-setup";
-
-const { login, password } = getTestingEnvironmentBrowser();
+setupEnv();
```
Comment (ThisIsMissEm) [inline e2e/browser/test/e2e.playwright.ts:29]: Why doesn't `getBrowserTestingEnvironment` invoke `setupEnv()` ?

Comment (chelseapinka) [reply e2e/browser/test/e2e.playwright.ts:29]: It does and this should be a dupe, will remove.

File: e2e/browser/test/globalSetup.ts [hunk 2]
```diff
@@ -19,11 +19,15 @@
...
+import {
+  setupEnv,
+  getBrowserTestingEnvironment,
+} from "@inrupt/internal-test-env";
 
 async function globalSetup() {
+  setupEnv();
   // Fail fast with dotenv:
-  getTestingEnvironmentBrowser();
+  getBrowserTestingEnvironment();
```
Comment (ThisIsMissEm) [inline e2e/browser/test/globalSetup.ts:30]: This shouldn't be needed here?

File: examples/solid-client-access-grants-demo/src/utils/getEnvironment.ts [hunk 3]
```diff
@@ -40,8 +40,8 @@ export function getEnvironment(): Environment {
       process.env.ACCESS_MANAGEMENT_APP ??
         "https://podbrowser.inrupt.com/privacy/access/requests/"
     ),
-    clientId: process.env.CLIENT_ID ?? "",
-    clientSecret: process.env.CLIENT_SECRET ?? "",
+    clientId: process.env.E2E_TEST_REQUESTOR_CLIENT_ID ?? "",
+    clientSecret: process.env.E2E_TEST_REQUESTOR_CLIENT_SECRET ?? "",
```
Comment (ThisIsMissEm) [inline examples/solid-client-access-grants-demo/src/utils/getEnvironment.ts:44]: I don't think this is correct? I think the example has (or should have) it's own `.env` file

**Review by ThisIsMissEm — DISMISSED** [review]
A few minor changes, but otherwise looks good

**Comment by ThisIsMissEm** [comment]
Looks like the E2E is mostly failing because it's reusing a container name between test units, one test is failing, and it's cascading to the others by the looks. https://github.com/inrupt/solid-client-access-grants-js/actions/runs/3470363521/jobs/5798473800

**Review by NSeydoux — APPROVED** [review]
Looks good!

**Pull request merged**